contrib_bots/lib/DefineBot/docs.md

```diff
@@ -0,0 +1,18 @@
+# DefineBot
+
+This is a bot that defines a word that the user inputs.
```

Comment:
In the docs you only write about words, not phrases. What about the instruction [here](https://github.com/zulip/zulip/pull/2757/files#diff-b99a7a8bf879fbff48ba61acc1943e16R39)?